src/Apps/Legion/Hooks/useEntities.tsx

```diff
@@ -97,6 +117,8 @@ export const useEntities = () => {
          * Callback to delete the entity from state.
          * NOTE: this is not a deep update. It will only delete the root level element
          */
-        deleteEntity: deleteEntity
+        deleteEntity: deleteEntity,
```

Comment:
we can just use the key names since they have same names with values variables.
```suggestion
        deleteEntity,
```